<launch> 
	<node pkg="offboard_sample" type="offb_att" name="offb_att" output="screen">
		<!--<remap from="~pos_cmd" to="/planning/pos_cmd"/>-->
		<param name="takeoff_height" value="2"/>
		<param name="flag_simulation" value="true"/>
		<remap from="~pos_cmd" to="/planning/pos_cmd"/>
		<rosparam file="$(find offboard_sample)/cfg/ctrl_param_fpv.yaml" command="load" />
	</node> 
	<node pkg="offboard_sample" type="keyboard.py" name="keyboard"  />
	
</launch>
                                                                                                                     
